save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/sarif/SarifFileUtils.kt

```diff
@@ -0,0 +1,52 @@
+/**
+ * Utility methods to work with SARIF files.
+ */
+
+package org.cqfn.save.plugin.warn.sarif
+
+import org.cqfn.save.core.files.parents
+
+import okio.FileSystem
+import okio.Path
+
+/**
+ * @return string with trimmed `file://` or `file:///`
+ */
+fun String.dropFileProtocol() = substringAfter("file://")
```

Comment:
where this comes from??

Reply:
This is just a regular URI protocol specification, which is used in absolute paths. I originally saw it in SARIF examples in official tutorials. As in https://github.com/analysis-dev/save/pull/347#discussion_r783031230, it's unlikely to face it in expected warnings, but may be useful for other applications